ml-agents/mlagents/trainers/trainer.py

```diff
@@ -151,19 +165,58 @@ def export_model(self):
         """
         self.policy.export_model()
 
-    def write_summary(self, global_step, lesson_num=0):
+    def write_training_metrics(self):
+        """
+        Write training metrics to a CSV  file
+        :return:
+        """
+        log_file = self.summary_path + '.csv'
+        with open(log_file, 'w') as f:
+            f.writelines('%s\n' % l for l in self.training_metrics)
+
+    def write_training_metric(self, delta_update_policy,
+                              delta_train_start):
+        """
+        Write training  metrics to be eventually be persisted in a  CSV
+        :param delta_update_policy: Time (in seconds) to update policy
+        :param delta_train_start: Time (in  seconds) since training started.
+        :return:
+        """
+        logger.debug("Time to update Policy: {:0.3f} s "
```

Comment:
What do you think about adding newlines after each of these metrics (e.g. like in RLLib) and including a header with what training step they refer to?

Reply:
PTAL